plugins/available/autojump.plugin.bash

```diff
@@ -1,8 +1,13 @@
 cite about-plugin
 about-plugin 'Autojump configuration, see https://github.com/wting/autojump for more details'
 
-# Only supports the Homebrew variant at the moment.
+# Only covers MacOS brew and Debian installs
 # Feel free to provide a PR to support other install locations
 if command -v brew &>/dev/null && [[ -s $(brew --prefix)/etc/profile.d/autojump.sh ]]; then
-  . $(brew --prefix)/etc/profile.d/autojump.sh
+  source $(brew --prefix)/etc/profile.d/autojump.sh
+fi
+
+# Debian / Ubuntu - install it via apt-get install autojump
+if [[ -s /usr/share/doc/autojump/README.Debian ]]; then
```

Comment:
I think it's safe to check for the actual script. If you could change that check, I'll merge...